styles.css

```diff
@@ -610,3 +626,486 @@ li {
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 1472.8px;
+    height: 1434.3px;
+    position: relative;
+    top: -28px;
+    right: 30px;
+  }
+
+  .bar {
+    position: relative;
+    width: 1440px;
+    height: 72px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    align-content: center;
+    list-style: none;
+    margin-top: -40px;
+    right: 5%;
+  }
+
+  .nav_bar ul {
+    display: flex;
+    text-decoration: none;
+    align-items: center;
+    align-content: center;
+    justify-content: center;
+    text-align: center;
+    gap: 30px;
+    margin-left: 40px;
+  }
+
+  .nav_bar a {
+    width: 63px;
+    height: 20px;
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 150px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    color: #344563;
+  }
+
+  .h1 {
+    width: 920px;
+    height: 114px;
+    font-family: 'Inter', sans-serif;
+    font-style: none;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    text-align: center;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+    padding-top: 60px;
+  }
+
+  .supporting {
+    width: 780px;
+    height: 96px;
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 32px;
+    text-align: center;
+    color: #172b4d;
+    position: relative;
+    left: 60px;
+  }
+
+  #button_social {
+    width: 200px;
+    height: 40px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    list-style: none;
+    position: relative;
+    bottom: 20px;
+    left: -30px;
+  }
+
+  #vector {
+    display: contents;
+    border: none;
+    border-color: none;
+    background: none;
+  }
+
+  #vector img {
+    padding: 8px;
+    background: #fff;
+    border-radius: 32px;
+    position: relative;
+    bottom: 25px;
+    right: 15px;
+    border: transparent;
+  }
+
+  /* work section */
+
+  .work {
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    padding: 64px 24px;
+    position: absolute;
+    top: 1059px;
+    left: 115px;
+    width: 1171px;
+    height: 1000px;
+    display: inline-grid;
+    grid-template-columns: auto auto auto;
+    gap: 25px;
+    margin-top: 200px;
+  }
+
+  .work_title {
+    width: 500px;
+    height: 48px;
+    font-family: 'inter', sans-serif;
+    font-size: 30px;
+    font-weight: 700;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    color: #172b4d;
+    position: relative;
+    left: 33%;
+    bottom: 10%;
+  }
+
+  .card {
+    width: 380px;
+    height: 498px;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+    background: #ebf0ee;
+  }
+
+  .card_inside {
```

Comment:
The background of the card content should be white and technologies should be in pills as per the Figma design. Please consider fixing this issue. 👍🏽 

Your Design
<img width="372" alt="Screenshot 2023-01-21 at 9 21 10 AM" src="https://user-images.githubusercontent.com/24832846/213844271-bb2d449a-013a-483d-b857-e9432d9c79ad.png">

Figma Template
<img width="375" alt="Screenshot 2023-01-21 at 9 21 27 AM" src="https://user-images.githubusercontent.com/24832846/213844275-27d6c579-8c82-4813-83d2-a91a7d9155a9.png">